packages/webview_flutter/webview_flutter_platform_interface/lib/src/types/http_response_error.dart

```diff
@@ -38,8 +41,16 @@ class HttpResponseError {
   /// Used by the platform implementation to create a new [HttpResponseError].
   const HttpResponseError({
     required this.statusCode,
+    this.requestParams,
+    this.response,
   });
 
   /// The HTTP status code.
   final int statusCode;
+
+  /// The associated request parameters.
+  final LoadRequestParams? requestParams;
```

Comment:
Isn't this the arguments object to a platform interface method? I would think we would want a new object that had just the things we wanted to provide to clients of the error handler, rather than using an object that will contain everything we might ever add to an unrelated method.

@bparrishMines Is there a compelling reason to re-using this object this way?

Reply:
I assumed it would be redundant since they would essentially have all the same parameters besides `body`. But this could be restrictive in the future. I agree that it would probably be better to create a separate class.